src/cascadia/TerminalControl/SearchBoxControl.xaml

```diff
@@ -3,15 +3,15 @@
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-    xmlns:local="using:Microsoft.Terminal.TerminalControl"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     mc:Ignorable="d"
     x:Name="Root"
     HorizontalAlignment="Stretch"
     VerticalAlignment="Top"
     d:DesignHeight="55"
     d:DesignWidth="285"
-    TabNavigation="Cycle">
+    TabNavigation="Cycle"
+    IsTabStop="True">

```

Comment:
I don't think we actually need `IsTabStop` here. It isn't a tab stop from anything.

Reply:
"IsTabStop" means "this control, or a control inside it, can receive focus."